engine/seq/base.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, name: str = "", func=None):
     def reset(self) -> None:
         pass
 
-    def handle_input(self, input: str) -> None:
+    def handle_input(self, input: str) -> None:  # noqa: A002
```

Comment:
"Argument `input` is shadowing a Python builtin"
i'm not sure what this function does so i noqa'd it for now

Reply:
this should probably be fixed if its something on our end we can change
https://docs.python.org/3/library/functions.html#input

Reply:
Should be simple enough to just rename the parameter to something else, we have full control over that.

Reply:
As for what it does, the function probably isn't necessary anymore. In the Evoland code, it allowed for capturing a character from the keyboard in `SequencerEngine` and pass it along to the current state, allowing you to pass user input to it. I don't remember if I ever used it for anything (though it's possibly useful for semi-manual debug states), and it shouldn't ever be used in the "real" run.

With `imgui`, we can probably do this in a different way, so the `handle_input` function becomes redundant and should be removed.